### ISSUE TITLE
Are scheduled tasks lost when consumer quits?

### ISSUE THREAD

**Issue by john-bonachon** [coleifer/huey#225]
Say I have huey consumer running on my computer, and then I turn it off, or just stop the consumer. Are scheduled tasks lost?

**Comment by coleifer** [comment]
Good question. So, huey tries to keep as little stuff in memory while working, so things like un-consumed tasks, or tasks sitting in the schedule until a future time, will be managed by the storage engine. It's possible, but unlikely, that huey could read from the schedule and be killed before it has time to enqueue the scheduled tasks...but that's quite unlikely since the interval between reading and enqueueing is very small.

So, long story short, huey will persist scheduled and enqueued tasks from execution to execution.

**Comment by john-bonachon** [comment]
Thanks for replying
